packages/components/slider/src/slider.vue

```diff
@@ -229,6 +228,13 @@ const precision = computed(() => {
 
 const { sliderWrapper } = useLifecycle(props, initData, resetSize)
 
+useEventListener(sliderWrapper, 'touchstart', onSliderWrapperPrevent, {
+  passive: false,
+})
+useEventListener(sliderWrapper, 'touchmove', onSliderWrapperPrevent, {
+  passive: false,
+})
+
```

Comment:
It can be merged here, `useEventListener` supports setting multiple events.

There is another question. To prevent SSR errors, maybe we should put the event binding logic in `onMounted` hook. 🤔